bin/casimir.js

```diff
@@ -8,10 +8,11 @@ properties.server.favicon = __dirname + '/../' + properties.server.favicon
 properties.server.compression = process.env.COMPRESSION || properties.server.compression
 properties.engine.view_folder = __dirname + '/../' + properties.engine.view_folder
 properties.engine.static_folder = __dirname + '/../' + properties.engine.static_folder
+properties.debug = process.env.DEBUG === 'true'
```

Comment:
now possible from [this commit](https://github.com/Colored-Coins/Colored-Coins-Block-Explorer/pull/8/commits/3cc6bb5fb40a9bc3e620b55db2df866a2af676c3)